### ISSUE TITLE
Mixin-name-pattern is not working for postcss-mixins

### ISSUE THREAD

**Issue by sourav-kumar** [stylelint/stylelint#2600]
> A feature request?

e.g. "A bug where..."

> Which rule, if any, is this issue related to?

Mixin-name-pattern

> What CSS is needed to reproduce this issue?
```
@define-mixin mixinname $abc $acb {}
@mixin mixinname $ab $bc;
```

> What stylelint configuration is needed to reproduce this issue?

e.g.

using stylelint-scss
```

> Which version of stylelint are you using?

e.g. `6.7.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

e.g. "CLI with `stylelint --config myconfig *.css`"

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

e.g. "Yes, it's related to SCSS maps."

> What did you expect to happen?

e.g. "No warnings to be flagged."

> What actually happened (e.g. what warnings or errors you are getting)?

e.g. "The following warnings were flagged:"

```shell
test.css
1:21    Expected 2 spaces (indentation)
```

**Comment by evilebottnawi** [comment]
@sourav-kumar Please update stylelint to latest version

**Comment by sourav-kumar** [comment]
@evilebottnawi , this is a stylelint-scss rule, so i am using updated version of stylelint and stylelint-scss. But when I am going to validate postcss mixin syntax ,
```
define-mixin mixinname $a $b {}

or 

@mixin $c $d;
```
its throwing error for rule 'mixin-name-pattern.

Please check once, if you get chance.

Thank You,
Sourav

**Comment by jeddy3** [comment]
Dupe of https://github.com/kristerkari/stylelint-scss/issues/121

@sourav-kumar [`at-mixin-pattern`](https://github.com/kristerkari/stylelint-scss/tree/master/src/rules/at-mixin-pattern) comes from the 3rd party `stylelint-scss` plugin. stylelint itself is only focused on standard CSS syntax.

As an aside, enforcing a pattern of `@define-mixin` might not be the domain of `stylelint-scss` either, as that plugin focuses on _SCSS-like_ syntax and constructs (and `@define-mixin` is not found in SCSS). As such, I'd recommend [creating your own plugin](https://stylelint.io/developer-guide/plugins/), called `at-define-mixin-pattern`, which would specifically enforce a pattern for `@define-mixin`.

**Comment by sourav-kumar** [comment]
@jeddy3 , thank you for your response.
